package.json

```diff
@@ -39,13 +39,18 @@
     "eslint": "4.8.0",
     "eslint-config-airbnb": "15.1.0",
     "eslint-plugin-import": "2.7.0",
+    "eslint-plugin-jasmine": "2.10.1",
     "eslint-plugin-jsx-a11y": "5.1.1",
+    "eslint-plugin-mocha": "5.2.0",
     "eslint-plugin-react": "7.4.0"
   },
   "dependencies": {
+    "base-64": "0.1.0",
```

Comment:
There is no other library that DOES support bigints?

Reply:
We tried `json-bigint`, but that was not able to read the JSON structure. Not even when the option was selected to store bigints as strings. So eventually I resorted to manually selecting the structure.

Reply:
Nope, however as I earlier noted, base64 is available natively in NodeJS, to base-64 isn't needed.